swan-lake/by-example/graphql-mutations/content.jsx

```diff
@@ -24,21 +24,21 @@ service /graphql on new graphql:Listener(4000) {
         self.person = new("Walter White", 51);
     }
 
-    // A resource function represents a field in the root \`Query\` operation.
-    resource function get profile() returns Person {
+    // A resource method represents a field in the root \`Query\` operation.
+    resource method get profile() returns Person {
 
         return self.person;
     }
 
-    // A remote function represents a field in the root \`Mutation\` operation. After updating the
+    // A remote method represents a field in the root \`Mutation\` operation. After updating the
     // name, the \`person\` object will be returned.
-    remote function updateName(string name) returns Person {
+    remote method updateName(string name) returns Person {
         self.person.setName(name);
         return self.person;
     }
 
-    // Remote function to update the age.
-    remote function updateAge(int age) returns Person {
+    // remote method to update the age.
```

Comment:
```suggestion
    // Remote method to update the age.
```

Reply:
Need to change in the BBE source in the dist repo during the BBE hackathon.